server/livecheck.py

```
@@ -17,11 +17,47 @@
 -----END CERTIFICATE-----
 """
 
+root_cert_text_dev = """
+-----BEGIN CERTIFICATE-----
+MIIBtjCCAV2gAwIBAgIUD5z+lzBU64i8+7ZarbmofAk9yjAwCgYIKoZIzj0EAwIw
+MDELMAkGA1UEBhMCVVMxCzAJBgNVBAgMAkNBMRQwEgYDVQQKDAtWb3RpbmdXb3Jr
+czAgFw0yNDEwMjgwNDExMzVaGA8yMTI0MTAwNDA0MTEzNVowMDELMAkGA1UEBhMC
+VVMxCzAJBgNVBAgMAkNBMRQwEgYDVQQKDAtWb3RpbmdXb3JrczBZMBMGByqGSM49
+AgEGCCqGSM49AwEHA0IABJDLSJQlKgCPZSgP+ZVpgareR1KWWRd4FjR94JymY21Q
+AcxLmgCD9jWnq+lp6SVy1Dz/NJ6Au23oRNgvWqGi8emjUzBRMB0GA1UdDgQWBBRM
+c0o7/6216xmY8apne2R8nQnmrTAfBgNVHSMEGDAWgBRMc0o7/6216xmY8apne2R8
+nQnmrTAPBgNVHRMBAf8EBTADAQH/MAoGCCqGSM49BAMCA0cAMEQCIG1WBl7HOeDB
+ftdu7Xmh0G4kzGktQ5hkJeOzRXNn346bAiB6QIjc1c61pM+E/MME4TXdOgmaS1gt
+QgBhqrQsjg5eTQ==
+-----END CERTIFICATE-----
+"""
+
+VX_IANA_ENTERPRISE_OID = '1.3.6.1.4.1.59817'
+
+VX_CUSTOM_CERT_FIELD = {
+    "MACHINE_ID": f"{VX_IANA_ENTERPRISE_OID}.6",
+}
+
 def makeTempFile(content, mode="w", encoding="utf-8"):
     tf = tempfile.NamedTemporaryFile(mode=mode, encoding=encoding, delete=False)
     tf.write(content)
     return tf.name
 
+def parseCertDetails(cert):
```

I think it's time to add some tests here, especially for this function.

I would also separate out the function that parses the extracted subject line. This is where security goblins live, so having a clear function with some clear tests and fail fast will be good.

---

[Commit](https://github.com/votingworks/livecheck-web/pull/2/commits/a996f13baa5be6508b96462a6924ab3061384e41)

1. Separated out `parseCertSubject` into it's own function
2. First pass at tests added, @benadida could you let me know if there is anything additional that should be covered?